eZ/Publish/Core/Persistence/Legacy/Content/Gateway/DoctrineDatabase/QueryBuilder.php

```diff
@@ -213,13 +217,10 @@ public function createVersionInfoFindQuery()
             $this->dbHandler->aliasedColumn($query, 'published', 'ezcontentobject'),
             $this->dbHandler->aliasedColumn($query, 'status', 'ezcontentobject'),
             $this->dbHandler->aliasedColumn($query, 'name', 'ezcontentobject'),
-            $this->dbHandler->aliasedColumn($query, 'language_mask', 'ezcontentobject'),
-            // Content object names
-            $this->dbHandler->aliasedColumn($query, 'name', 'ezcontentobject_name'),
-            $this->dbHandler->aliasedColumn($query, 'content_translation', 'ezcontentobject_name')
+            $this->dbHandler->aliasedColumn($query, 'language_mask', 'ezcontentobject')
         )->from(
             $this->dbHandler->quoteTable('ezcontentobject_version')
-        )->leftJoin(
+        )->innerJoin(
```

Comment:
@pspanja I didn't see any reason why this needs to be leftJoin, hence the change here.